### PR TITLE
modified doc

### PR DESCRIPTION
**Pull request by keikhara**
@Microsoft/omsagent-devs @amitsara 
Modified Docker Instruction. Please check. 
Added setting for Upstart, systemd, and Amazon Linux.

### CODE REVIEW

File: docs/Docker-Instructions.md [hunk 1]
```diff
@@ -10,8 +10,7 @@ With this feature, you'll be able to:
 
 Because this is a private preview, we'll need your OMS workspace ID to enable it. If you aren't included in the private preview and are interested in joining, drop us a line at OMSContainers@microsoft.com.
 
-## Setting up
-Your container hosts must be running:
+### Supported Linux Operating Systems and Docker:
```
Comment (agup006) [inline docs/Docker-Instructions.md]: Is Ubuntu not supported?

File: docs/Docker-Instructions.md [hunk 2]
```diff
@@ -21,16 +20,47 @@ And one of:
...
 * SUSE Linux Enterprise Server 12
 
-You'll need to [install the OMS Agent for Linux] (docs/OMS-Agent-for-Linux.md#steps-to-install-the-oms-agent-for-linux) on each host and then do the following to configure your containers to use the FluentD logging driver:
+## Setting up
+As a pre-requisite, docker must be running prior to this OMS Linux Agent installation. If you have installed before running docker, please re-install OMS Agent. For more information about docker, please go to https://www.docker.com/.
+
+The following settings for container host must be done to monitor the containers. 
 
- * Edit `/etc/default/docker` and add this line:
+#### Settings on container host - systemd
```
Comment (agup006) [inline docs/Docker-Instructions.md]: Might be helpful to indicate which OSes are systemd, and which are upstart

File: docs/Docker-Instructions.md [hunk 3]
```diff
@@ -21,16 +20,47 @@ And one of:
...
+Make sure you add $DOCKER_OPTS in "ExecStart=/usr/bin/docker daemon" within your docker.service file.
+example)
+```
+[Service]
+Environment="DOCKER_OPTS=--log-driver=fluentd --log-opt fluentd-address=localhost:25225"
+ExecStart=/usr/bin/docker daemon -H fd:// $DOCKER_OPTS
+```
+- Restart docker service.
+```
+systemctl restart docker.service
```
Comment (agup006) [inline docs/Docker-Instructions.md]: should add `sudo`

Comment (agup006) [inline docs/Docker-Instructions.md]: Additionally, should add `service docker restart`

File: docs/Docker-Instructions.md (hunk 1 above)
Comment (keikhara) [inline docs/Docker-Instructions.md]: It should be supported but we will update more of the Linux OS settings after we tested our matrix list.

**Comment by agup006** [comment]
Should also add different email from OMSContainer from Issue https://github.com/Microsoft/OMS-Agent-for-Linux/issues/138

File: docs/Docker-Instructions.md (hunk 3 above)
Comment (keikhara) [inline docs/Docker-Instructions.md]: let me add sudo.

**Comment by keikhara** [comment]
It seems like if you send it omscontainer email address this comes up. We may need to fix that. Not sure who to talk to…

File: docs/Docker-Instructions.md (hunk 2 above)
Comment (keikhara) [inline docs/Docker-Instructions.md]: I did add them now. Normally, docker does not even add OS and versions to these settings.

**Comment by keikhara** [comment]
@Microsoft/omsagent-devs @amitsara 
Hi all, 

I did the updates. If you are all find with the doc changes, I would like to merge this since this may affect some private preview customers. 

regards, 

Keiko

**Comment by keikhara** [comment]
@Microsoft/omsagent-devs @amitsara 

Anurag, please sign-off on this doc. 

thanks,

Keiko

**Comment by agup006** [comment]
sign off

**Pull request merged**